odoo/tests/common.py

```diff
@@ -1735,6 +1735,17 @@ class HttpCase(TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.bert_user = new_test_user(
+            cls.env,
+            login='bert_user',
+            name='Bert User',
+            email='user@example.com')
+        cls.hugo = new_test_user(
+            cls.env,
+            login='hugo_user',
+            name='Hugo',
+            email='hugo@example.com',
+            tz='UTC')
```

Comment:
This should be in the hr_timesheet because, without the hr_timesheet, it will be useless.